test/Anchor/getLatestAnchorInfo.js

```diff
@@ -0,0 +1,73 @@
+const chai = require('chai');
+const Web3 = require('web3');
+const sinon = require('sinon');
+const Anchor = require('../../src/ContractInteract/Anchor');
+const SpyAssert = require('../../test_utils/SpyAssert');
+
+const { assert } = chai;
```

Comment:
You could do directly:
```suggestion
const { assert } = require('chai');
```

Reply:
Changed it in all the test files.